sw/device/lib/dif/dif_entropy_src.h

```diff
@@ -371,6 +408,18 @@ typedef enum dif_entropy_src_result {
    * Indicates that entropy is not idle
    */
   kDifEntropySrcNotIdle = 5,
+  /**
+   * Indicates an invalid firmware override configuration.
+   */
+  kDifEntropySrcInvalidFwOverrideSettings = 6,
+  /**
+   * Indicates invalid firmware observe FIFO interrupt threshold.
+   */
+  kDifEntropySrcInvalidFwOverrideBufferThreshold = 7,
+  /**
+   * Indicates invalid firmware FIFO override read size.
+   */
+  kDifEntropySrcInvalidFifoReadLen = 8,
 } dif_entropy_src_result_t;
```

Comment:
This type has been deprecated in #8412 as well in favor of the globally defined one in `sw/device/lib/dif/dif_base.h`. 

My guess is that instead of `kDifEntropySrcInvalidFwOverrideSettings` you may just use `kDifBadArg` and instead of `kDifEntropySrcInvalidFwOverrideBufferThreshold` or  `kDifEntropySrcInvalidFifoReadLen` you may just use `kDifIpFifoFull` (also defined in `sw/device/lib/dif/dif_base.h`). Applies throughout.

Additionally, we did just have a brief conversation on these in #8388, so maybe we need to adjust what is defined in `sw/device/lib/dif/dif_base.h`, but this suggestion is just going off what is currently defined.

Reply:
Done.